test: isolate command and arguments in use of subprocess.run

PaulWay opened the pull request:
The command and its arguments are the most important part of the arguments to `subprocess.run`, so they should be highlighted in the indentation.

This was originally going to be a trivial spelling fix as a test of the process of contributing to `osbuild`, but your comments and docstrings are all impeccably spelled!

Signed-off-by: Paul Wayper <paulway@redhat.com>

PaulWay commented:
Just to clarify, this was asked for by Stef Walter as part of a test to see how easy it was to find and contribute to this project.

gicmo commented:
Happy to take this; I personally quite like when the command in the `subprocess.run` call is isolated and thus more clearly visible. This needs a rebase on main though to make CI happy (github's lack of merge queue allowed two commits that were green to be merged but the one modified the test data and then the test-data regeneration check was broken on main).

The commit message (subject) is a bit meta. Maybe something like `test: isolate command for subprocess.run` or something.

gicmo commented:
> Just to clarify, this was asked for by Stef Walter as part of a test to see how easy it was to find and contribute to this project.

Would love to get feedback how easy it indeed was! We provide a VS Code dev container to be able to quickly hack on this project but I am sure we can make it better for newcomers somehow.

PaulWay commented:
Hi @gicmo - thanks for that.
Well, maybe I'm not a typical developer, since my basic way of finding and contributing to this repository was to find it in Github and then fork it, clone it, create a branch, write the code, push it to my fork and create a PR from that.  I didn't need to set up an environment, I already have my development environment set up :smile:

dvdhrm commented:
> You can reject this if you like, since it seems to be formatted differently from your regular layout.

Yeah, I like those splits, too! Looks good to me (but see comments from @gicmo).
 
> I tried looking for a spelling error in your comments and docstrings but it's all impeccably spelled!

:heart:

PaulWay commented:
Thanks for that - I've updated this to do the same for a few other uses of subprocess.run in the test file.  I've also updated the title and description now that it's actually useful.

teg reviewed (approved):
Nice! Thanks Paul!

teg commented:
Looks like we have found room for improvement for external contributors, our full CI is not getting triggered, and we don't currently have a nice work-around. As pointed out by @ondrejbudai in chat, we need something like https://github.com/rhinstaller/anaconda/blob/master/.github/workflows/tests-contributors.yml

For now force-merging after double checking the code (as it is just whitespace and passes the unit tests and linter).
